### PR TITLE
Adding TableView/useTable/useTableState docs mdx

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes https://github.com/adobe/react-spectrum/issues/1648 and #1913 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-spectrum/table/docs/TableView.mdx [hunk 1]
```diff
@@ -85,43 +85,40 @@ which dramatically improves performance. Make sure that each rendered property i
...
-
-  let rows = [
-    {id: '1', name: 'Games', date: '6/7/2020', type: 'File folder'},
-    {id: '2', name: 'Program Files', date: '4/7/2021', type: 'File folder'},
-    {id: '3', name: 'bootmgr', date: '11/20/2010', type: 'System file'},
-    {id: '4', name: 'log.txt', date: '1/18/2016', type: 'Text Document'}
-  ];
-
-  return (
-    <Flex flexGrow={1} maxWidth="size-6000">
```
Comment (LFDanLu) [inline packages/@react-spectrum/table/docs/TableView.mdx]: Do we wanna remove the Flex wrapper from the rest of the examples for consistency?

**Review by LFDanLu — COMMENTED** [review]
Thanks for all the updates, looks really good to me. Just two really small things noted

File: packages/@react-spectrum/table/docs/TableView.mdx (hunk 1 above)
Comment (devongovett) [reply packages/@react-spectrum/table/docs/TableView.mdx]: yeah I think Danni hasn't pushed her changes yet.

**Comment by dannify** [comment]
To be merged after release

**Pull request merged**